components/invitation_modal/overage_users_banner_notice/index.tsx

```diff
@@ -85,15 +98,24 @@ const OverageUsersBannerNotice = () => {
                     id='licensingPage.overageUsersBanner.noticeDescription'
                     defaultMessage='Notify your Customer Success Manager on your next true-up check. <a>Contact Sales</a>'
```

Comment:
Nit: Since we are setting the inner link text dynamically, I tihnk we should change the defaultMessage and i18n/en.json string to say `... check. <a></a>`